script/foreman-debug

```diff
@@ -246,9 +246,18 @@ else
 fi
 printv "Determined $OS distribution"
 
+clean_temp() {
+  [ "$NOTAR" -eq 1 ] && exit 0
+  # prevent from removing anything user-defined
+  [[ "$DIR" != /var/tmp* ]] && exit 0
+  printv "Cleaning $DIR"
+  rm -rf $DIR
```

Comment:
Missing quotes `"$DIR"`, just to be on a safe side

Reply:
Added.